pkgs/by-name/op/openpgp-card-tools/package.nix

```diff
@@ -37,6 +38,13 @@ rustPlatform.buildRustPackage rec {
     };
   };
 
+  postInstall = ''
```

Comment:
Please make this conditional on `stdenv.buildPlatform.canExecute stdenv.hostPlatform`, otherwise it breaks cross compilation.

Reply:
Thank, I did not think about that.  Done.